src/components/Profile/ProfileEditButtonGroup.jsx

```diff
@@ -26,8 +27,7 @@ const ProfileEditButtonGroup = ({ edit, handleCancelEdit, handleEditInput }) =>
   <Box
     sx={{
       display: 'flex',
-      gap: '10px',
```

Comment:
While removing the `marginTop` seems good since we are moving the button group to a little icon in the corner, think we'll still need the gap between buttons. At the moment the buttons are sticking together without the gap.